File: 682-baseball-game/baseball-game.py
class Solution:
    def calPoints(self, operations: List[str]) -> int:
        l = []
        for i in operations:
            n = len(l)
            if i == 'C':
                l.pop()
            elif i == 'D':                
                a = l[n - 1] * 2
                l.append(a)
            elif i == '+':
                l.append(l[n - 1] + l[n - 2])
            else:
                l.append(int(i))
        return sum(l)

File: 682-baseball-game/baseball-game.java
class Solution {
    public int calPoints(String[] operations) {
        ArrayList<Integer> arr = new ArrayList<>();
        for(String c : operations){
            int n = arr.size();
            if(c.equals("C")){
                arr.remove(n-1);
            }else if(c.equals("D")){
                arr.add(arr.get(n - 1) * 2);
            }else if(c.equals("+")){
                arr.add(arr.get(n - 1) + arr.get(n - 2));
            }else{
                arr.add(Integer.parseInt(c));
            }
        }
        int sum = 0;
        for(int i = 0; i < arr.size(); i++){
            sum += arr.get(i);
        }
        return sum;
    }
}